### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Encrypted messages can be opened in owa. Only in outlook, the automapping feature is required.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5217

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -247,7 +247,7 @@ sections:
       - question: |
           Can I open encrypted messages sent to another user's mailbox with Fullaccess?
         answer: |
-          Users can open encrypted messages as long as they are given direct access and automapping is turned ON. Access is not allowed if the access is granted via an email-enabled security group.
+          Users can open encrypted messages in outlook as long as they are given direct access and automapping is turned ON. Access is not allowed in outlook if the access is granted via an email-enabled security group. The users can access encrypted messages through Outlook on the web.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Users can open encrypted messages in Outlook as long as they are given direct access and automapping is turned ON. Access is not allowed in Outlook if the access is granted via an email-enabled security group. Users can access encrypted messages using Outlook on the web.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder, please merge this PR. Thanks

**Comment by KCCross** [comment]
@msbemba Please resolve - You can sign in to the OME portal to retrieve mail as long as the sender's organization is active and the mail has not been configured to expire.

**Comment by msbemba** [comment]
Thank you @KCCross

**Pull request closed without merging**